services/web/src/screens/Settings/Menu.js

```diff
@@ -1,30 +1,31 @@
-import React from 'react';
-import { Menu } from 'semantic';
-import { NavLink } from '@bedrockio/router';
+import { IconFingerprintScan, IconUser } from '@tabler/icons-react';
+import PageHeader from 'components/PageHeader';
 
-import Breadcrumbs from 'components/Breadcrumbs';
+export default function SettingsMenu() {
+  const items = [
```

Comment:
I feel like I'm seeing `items` as an array everywhere... it's kind of distracting and breaks the flow